fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/provider/impl/AzureProvider.java

```diff
@@ -498,4 +500,19 @@ public void registerTransient(long executionId, ExportTransientUserData transien
         this.fhirResourceType = fhirResourceType;
         this.chunkData = transientUserData;
     }
-}
+
+    @Override
+    public void pushEndOfJobOperationOutcomes(ByteArrayOutputStream baos, String folder, String fileName)
+            throws FHIRException {
+        String fn = folder + File.separator + fileName;
```

Comment:
does the azure blob client work with window's file separator?

Reply:
Good catch... it would have only hit on Windows.